pagopa/io-app
# fix: [IOBP-1100] German FAQ strings

## Short description
This pull request corrects the display strings for the German FAQ.

## List of changes proposed in this pull request
- Update `io-service-metadata` version
- Add german key to `localeToLocalizedMessageKey` constant
- Added `fallbackForLocalizedMessageKeys` function to provide a fallback for German translations when they are not available
- Updated locale handling to use the new fallback mechanism

## How to test
> [!IMPORTANT]
> Align `io-dev-api-server` with this [PR](https://github.com/pagopa/io-dev-api-server/pull/453) 
- Select German language, reload the app
- Open the app
- Tap on FAQ
- Ensure that categories and subcategories are in german

## Preview

https://github.com/user-attachments/assets/333858a3-d03a-496e-8869-e28f10d2843c
